spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1155,6 +1197,241 @@ public Void run(TransactionContext transaction) throws Exception {
   }
   // [END spanner_dml_batch_update]
 
+  // [START spanner_create_table_with_datatypes]
+  static void createTableWithDatatypes(DatabaseAdminClient dbAdminClient, DatabaseId id) {
+    OperationFuture<Void, UpdateDatabaseDdlMetadata> op =
+        dbAdminClient.updateDatabaseDdl(
+            id.getInstanceId().getInstance(),
+            id.getDatabase(),
+            Arrays.asList(
+                "CREATE TABLE Venues ("
```

Comment:
Probably ok for the sample, but might make testing difficult at times.  (using a dynamic table name would be better)